sharepoint-adfs/azuredeploy.json

```diff
@@ -359,7 +499,7 @@
       "apiVersion": "2019-09-01",
       "type": "Microsoft.Network/networkSecurityGroups",
       "name": "[variables('networkSettings').nsgSubnetDCName]",
-      "location": "[parameters('location')]",
+      "location": "[resourceGroup().location]",
```

Comment:
need to revert these back to using the parameter...

Reply:
So the best practice is to use `parameters('location')` instead of `resourceGroup().location`?
I thought it was the opposite

Reply:
https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#parameters - essentially, use a parameter (so user can specify it independently) and default to [resourceGroup().location] for the common case.